vsintegration/src/FSharp.Editor/SignatureHelp.fs

```diff
@@ -155,17 +155,22 @@ type FSharpSignatureHelpProvider [<ImportingConstructor>]  (serviceProvider: SVs
 
         for method in methods do
             // Create the documentation. Note, do this on the background thread, since doing it in the documentationBuild fails to build the XML index
-            let methodDocs = XmlDocumentation.BuildMethodOverloadTipText(documentationBuilder, method.Description, true)
+            let methodDocs = XmlDocumentation.BuildMethodOverloadTipText(documentationBuilder, method.Description, false)
```

Comment:
I guess I should clarify - I believe that the right behavior here is the one that C# does (and that this PR does as well).  Showing all parameters could be too much on the eyes for any method with more than 3 parameters.  I don't have data proving this, but I believe that there are enough cases out there that justify this.